Tasks/AppCenterDistributeV1/Tests/L0.ts

```diff
@@ -166,5 +168,28 @@ describe('AppCenterDistribute L0 Suite', function () {
 
         done()
     });
+
+    it('Positive path: publish commit info (including commit message)', (done: MochaDone) => {
+        this.timeout(2000);
+
+        let tp = path.join(__dirname, 'L0PublishCommitInfo_1.js');
+        let tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+
+        tr.run();
+        assert(tr.succeeded, 'task should have succeeded');
+
+        done()
+    });
    
+    it('Positive path: publish commit info (excluding commit message)', (done: MochaDone) => {
+        this.timeout(2000);
+
+        let tp = path.join(__dirname, 'L0PublishCommitInfo_2.js');
+        let tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+
+        tr.run();
+        assert(tr.succeeded, 'task should have succeeded');
+
+        done()
```

Comment:
any reason to leave out ; after done()

Reply:
No reason - error from copy paste existing code (fixed those as well)